### ISSUE TITLE
Silent failure if the view starts with a space or newline

### ISSUE THREAD

**Issue by chancyk** [yysun/apprun#27]
AppRun seems to silently produce nothing if there's a space or newline before the first element in the view. I noticed this initially when using the backtick string form, e.g.:

```js
return `
<div>
  <div>hello</div>
  <div>world</div>
</div
`
```

The code below produces the issue for me on AppRun version 1.13.8 and Chrome 67:

```html
<html>
<head>
    <script src="https://unpkg.com/apprun@1.13.8/dist/apprun-html.js"></script>
</head>

<body>
    <div id="app-content">
    </div>

    <script>
        const state = {};
        const view = state => {
            return ' <div>hello</div>';
        };

        const update = {
        };
        app.start('app-content', state, view, update);
    </script>
</body>
</html>
```

**Comment by yysun** [comment]
Confirmed. I will release a fix tonight.

**Comment by yysun** [comment]
It has been fixed in 1.13.9

**Comment by chancyk** [comment]
Confirmed fixed. Thanks!
